node-api/backend/validator.go

```diff
@@ -80,7 +80,7 @@ func (f *validatorFilters) parseID(id string) {
 func (b Backend) FilteredValidators(
 	slot math.Slot, ids []string, statuses []string,
 ) ([]*beacontypes.ValidatorData, error) {
-	st, slot, err := b.stateFromSlot(slot)
+	st, resolvedSlot, err := b.stateFromSlot(slot)
 	if err != nil {
 		return nil, errors.Wrapf(err, "failed to get state from slot %d", slot)
 	}
```

Comment:
Noticed that the error message was getting overridden by the slot from state (stateFromSlot) instead of the slot passed in, hence renamed the var to `resolvedSlot` to correctly log the error message.